apps/bidsificator/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# syntax=docker/dockerfile:1
+FROM ubuntu:22.04
+SHELL ["/bin/bash", "-xe", "-o", "pipefail", "-c"]
+
+ARG APP_NAME
+ARG APP_VERSION
+
+WORKDIR /apps/${APP_NAME}
+
+ARG DEBIAN_FRONTEND=noninteractive
+ADD https://github.com/HIP-infrastructure/Bidsificator/archive/refs/tags/V${APP_VERSION}.zip .
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update -qy && \
+    apt-get install --no-install-recommends -qy \
+    unzip python3 python3-pip \
+    # lib necessary to have a working pyqt application \
+    libxcb-icccm4 libxcb-randr0 libxcb-render-util0 \
+    libxcb-shape0 libxcursor1 libxkbcommon-x11-0 \
+    libxcb-keysyms1 libglib2.0-0 libdbus-1-3 libxcb-icccm4 \
+    libxcb-randr0 libxcb-render-util0 libxcb-shape0 fontconfig \
+    libfontconfig1 libxcb-image0 libxcb-util1 && \
+    # unpack and put the app in the right place \ 
+    unzip V${APP_VERSION}.zip && \
+    mv Bidsificator-${APP_VERSION}/* . && \
+    rm -rf Bidsificator-${APP_VERSION} V${APP_VERSION}.zip && \
+    # install poetry project via pip \
+    pip install . && \
```

Comment:
Maybe you'll need git back again.

```bash
pip install --no-cache git+https://github.com/HIP-infrastructure/Bidsificator/@v${APP_VERSION}#egg=Bidsificator
```

Reply:
with an uppercase V, but it works nicely

Reply:
Big V? who does that?

Reply:
![image](https://github.com/user-attachments/assets/ffedba7c-c2ce-41f7-ac96-ab8981c2d13f)